tests/rbd_mirror/test_9500_shrink_img_at_secondary.py

```diff
@@ -0,0 +1,71 @@
+from tests.rbd.exceptions import IOonSecondaryError
```

Comment:
Not sure why `9500` is include but... you could rename the mode to `test-image-shrink.py`. Most of these seem to be workflows... you can have a single test module that covers these operations. This would allow you to have a separate workflow library and a test module... this way you can expand by adding only test flows with minor option changes to the test module.